pkgs/applications/virtualization/qemu/binfmt-p-wrapper.nix

```diff
@@ -5,18 +5,20 @@
 # The wrapper has to be static so LD_* environment variables
 # cannot affect the execution of the wrapper itself.
```

Comment:
Please check whether this comment is still applicable.

Reply:
How's that relevant to this PR?  We're still building a static executable, just not via pkgsStatic.

Reply:
Ah. I didn't realize adding that to buildInputs would create a static binary :+1:

Reply:
It doesn't, but passing `-static` to the compiler does. You have to do this whether using pkgsStatic or not, so no change is needed here.